src/e2e-app/mdc-tabs/mdc-tabs-e2e.ts

```diff
@@ -14,5 +14,4 @@ import {Component} from '@angular/core';
   templateUrl: 'mdc-tabs-e2e.html',
 })
 export class MdcTabsE2e {
-  // TODO: copy implementation from existing tabs e2e page.
```

Comment:
if there are no existing e2e tests for tabs we should probably just remove this test

Reply:
There are tests, it's just that the test component doesn't have any TS code. I've copied over all of the HTML and the tests themselves.